config.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+)
+
+// Config TODO
+type Config struct {
+	CustomerName string
+	Directory    map[string]interface{}
+	Slack        slackConfig
+	PagerDuty    pagerDutyConfig
+}
+
+// FromEnv TODO
+func (c *Config) FromEnv() error {
+	c.CustomerName = getVarFromEnv("CUSTOMER_NAME")
```

Comment:
You should decouple the way you consume environment variables from the actual configuration.

Your configuration struct should not concern itself by initializing from environment, instead let your CLI tool to take care of reading variables from CLI or Environment and then pass it to the structure like here:

https://github.com/gravitational/planet/blob/master/tool/planet/main.go#L79

It will allow you to avoid spreading non-constant variables and validation all over the code.

Reply:
TBD in future. Deferred now for time contrains.